src/bridge/graphs/peg_out.rs

```diff
@@ -824,8 +946,7 @@ impl PegOutGraph {
                         .unwrap()
                         .block_height
                         .is_some_and(|block_height| {
-                            block_height
-                                + self.start_time_timeout_transaction.num_blocks_timelock_1()
+                            block_height + self.connector_1.num_blocks_timelock_leaf_2
```

Comment:
I'm not opposed to this change but we should change it everywhere instead of just here. Now we have two conventions. Was there was reason for this change except that we felt it was better refactoring?

Reply:
This flowed naturally from the original design where we would ask the transaction object for `num_blocks_timelock_*`, which in turn asked the respective connector it encapsulated. Since we decoupled the connectors from the transactions, instead of asking the transactions for the value, we ask the respective connector. They were always the authoritative source of data for this.

We said we would complete the refactor for all the remaining connectors after this PR.